docs/src/model_setup/grids.md

```diff
@@ -90,7 +90,7 @@ example, to set up a 1D model with $N_z$ grid points
 julia> grid = RegularCartesianGrid(topology=(Flat, Flat, Bounded),
                             size=(1, 1, 90), extent=(1, 1, 1000))
 RegularCartesianGrid{Float64, Flat, Flat, Bounded}
-                   domain: x ∈ [0.0, 1.0], y ∈ [0.0, 1.0], z ∈ [-1000.0, 11.111111111111086]
+                   domain: x ∈ [0.0, 1.0], y ∈ [0.0, 1.0], z ∈ [-1000.0, -2.471452993948034e-14]
```

Comment:
hmmm

Reply:
Hmmm yeah it's not pretty but that's just floating point arithmetic I guess. Interestingly, in this case 
```julia
julia> grid = RegularCartesianGrid(topology=(Flat, Flat, Bounded),
                                   size=(10, 10, 10), x=(0, 1), y=(0, 1), z=(0, 1))
RegularCartesianGrid{Float64, Flat, Flat, Bounded}
                   domain: x ∈ [0.0, 1.0], y ∈ [0.0, 1.0], z ∈ [-1.6190752442450216e-17, 0.9999999999999999]
                 topology: (Flat, Flat, Bounded)
  resolution (Nx, Ny, Nz): (10, 10, 10)
   halo size (Hx, Hy, Hz): (1, 1, 1)
grid spacing (Δx, Δy, Δz): (0.1, 0.1, 0.1)
```
it's because
```julia
julia> -0.1 + 1.0 + 2*1.0 * 0.1
1.1

julia> -0.1 + (1.0 + 2*1.0 * 0.1)
1.0999999999999999
```
so we could probably fix it by changing
```julia
xF₊, yF₊, zF₊ = XF₊ = @. XF₋ + total_extent(topology, halo, Δ, L)
```
to something like
```julia
xF₊, yF₊, zF₊ = XF₊ = @. right_endpoint(topology, XF₋, halo, Δ, L)
```